### PR TITLE
[Docs](bangc-ops): Improved the API Guide for the release

### PR DESCRIPTION
**Pull request by GraceGuanH**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](../docs/MLU-OPS精度验收标准.md).

- [ ] diff1, diff1 <= 3e-3
- [ ] diff2, diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

|     No.      |           Details            |      Check Results       |
|----------------|---------------------------|---------------------|
|        1       |          Supported hardware         | MLU270 <br> MLU290 <br>MLU370|
|        2       |          Job types          |    block <br> U1 <br> U4    |
|        3       |         Layouts            |  NHWC 、NCHW、ARRAY etc    |
|        4       |         Whether multi-dimensions are supported              |                |
|        5       |          Whether element zero is supported             |               |
|        6       |         Data type(half/float)       |         half / float etc           |
|        7      |        Whether there is size limit           |             |

#### 3.1.3 New Feature Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test
- [ ] Multidimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test
- [ ] Nan/INF tests 
- [ ] For memory leak check details, see[GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](../docs/GTest-User-Guide-zh.md).
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md).

#### 3.1.4 Parameter Check

When a new operator is submitted, the test points are given and the test results are stated.

| Test Point         | Acceptance Criteria | Test Result (Error Message) |
| -------------- | -------- | -------------------- |
| Whether it conforms to the operator restriction | Normal error |                      |
| Whether illegal parameters are passed  | Normal error |                      |

### 3.2 Performance Test

See [MLU-OPS Performance Acceptance Criteria](../docs/MLU-OPS性能验收标准.md) for details.

Platform ：MLU270

|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-----|----|----|----|----|----|------|-----|
|op_name|   |    |     |    |    |    |     |
|op_name|   |    |     |    |    |    |     |

Platform ：MLU290

|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-----|----|----|----|----|----|------|-----|
|op_name|   |    |     |    |    |    |     |
|op_name|   |    |     |    |    |    |     |

Platform：MLU370

|Operator|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-----|----|----|----|----|----|------|-----|
|op_name|   |    |     |    |    |    |     |
|op_name|   |    |     |    |    |    |     |

### 3.3 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -219,7 +218,7 @@ struct mluOpContext;
  * The MLUOP context is created with ::mluOpCreate and the returned
  * handle should be passed to all the subsequent function calls.
  * You need to destroy the MLUOP context at the end with ::mluOpDestroy.
- * For more information, see "Cambricon MLUOP User Guide".
+ # For more information, see "Cambricon MLUOP User Guide".
```
Comment (wushaoqiang2168) [inline bangc-ops/mlu_op.h:221]: *号吧

Comment (GraceGuanH) [reply bangc-ops/mlu_op.h]: 按照沟通的，使用*#注释掉内容

**Review by wushaoqiang2168 — APPROVED** [review]

**Review by mahxn0 — APPROVED** [review]

**Review by DanieeelLiu — APPROVED** [review]

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -1304,8 +1326,8 @@ mluOpInitTensorSetMemberDescriptor(mluOpTensorSetDescriptor_t tensorSetDesc,
 /*!
  *  @brief Sets the position and scale factors used in fixed-point quantization.
  *  It is only used if you have quantized the input data with the symmetric
- *  fixed-point quantization with scale factor quantization method. For more
- *  information about quantization, see "Cambricon mluOp User Guide".
+ *  fixed-point quantization with scale factor quantization method. 
+ *# For more information about quantization, see "Cambricon mluOp User Guide".
```
Comment (duzekunKTH) [inline bangc-ops/mlu_op.h:1330]: 这是句尾多了一个空格？

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -1626,16 +1653,17 @@ mluOpStatus_t MLUOP_WIN_API mluOpGetGenerateProposalsV2WorkspaceSize(
...
+// Group:GenerateProposalsV2
 /*!
- *  @brief This operator is the second version of generate_proposals op to
- *    generate bounding box proposals for Faster Region-CNN.
- *    The proposals are generated for a list of images based on image
- *    score 'Scores', bounding box regression result 'BboxDeltas' as
- *    well as predefined bounding box shapes 'anchors'. Greedy non-maximum
- *    suppression is applied to generate the final bounding boxes.
+ *  @brief Generates bounding box proposals for Faster Region-CNN.
+ *  This operator is the second version of generate_proposals op.
```
Comment (duzekunKTH) [inline bangc-ops/mlu_op.h:1659]: 句尾空格删一下？

**Review by duzekunKTH — APPROVED** [review]

**Pull request merged**